### PR TITLE
[e2e] Added Namespace health and negative tests.

### PR DESCRIPTION
**Pull request by hhovsepy**
RFE https://github.com/kiali/kiali/issues/4826

Added tests for Health of Namespace for types: Apps, Workloads and Services.
Also added some negative tests.

### CODE REVIEW

**Comment by hhovsepy** [comment]
rebased

**Comment by hhovsepy** [comment]
Hi @nrfox , this PR is ready for review please.

File: tests/integration/tests/istio_config_test.go [hunk 1]
```diff
@@ -96,10 +96,20 @@ func TestIstioConfigDetails(t *testing.T) {
...
+}
+
 func TestIstioConfigPermissions(t *testing.T) {
 	assert := assert.New(t)
 	perms, err := utils.IstioConfigPermissions(utils.BOOKINFO)
 
 	assert.Nil(err)
 	assert.NotEmpty(perms)
+	assert.NotEmpty((*perms)[utils.BOOKINFO])
+	assert.NotEmpty((*(*perms)[utils.BOOKINFO])["authorizationpolicies"])
```
Comment (nrfox) [inline tests/integration/tests/istio_config_test.go:114]: This is an odd syntax that I've never seen before. I think it stems from the models themselves though and the fact that they are pointers to maps so nothing to change here.

**Review by nrfox — APPROVED** [review]
Coverage is looking good :+1:

**Review by Xunzhuo — APPROVED** [review]
Tests are looking good, thanks @hhovsepy

**Comment by hhovsepy** [comment]
thank you all for approval

**Pull request merged**